### DISCUSSION TITLE
Any workaround for restricted bluetooth availability?

### DISCUSSION THREAD

**Discussion by quaziqbal** [AppGround-io/bluetooth-keyboard-and-mouse-support#260] [Bugs & Problems]
First of all big kudos to you, I'm extremely slow typer on mechanical keyboard, yet I have fortunately cleared Amazon assessment test with the help of your app & SwiftKey keyboard. Now the problem is that after onboarding Amazon, the provided chrome os PC doesn’t permit me to install or modify anything. 

I have bought an external Bluetooth adapter as it was not pre-built in the pc. When I insert the bt adapter, the system recognizes it and start searching for Bluetooth devices nearby,  one and only one device it discovers & pairs with is my smart electric meter. I don't know why it doesn't recognize other Bluetooth enabled devices like phone, headset etc. I have changed various bt adapters but the story is same. 

My phone is rooted. I have managed to turn my device to detect it as USB HID through "USB GADGET TOOL" 

Is there any workaround to utilize your app in my condition of restricted bluetooth access?

**Comment by KRBTy** [comment]
Is it possible the external adaptor is using a very old version of BT it's unable to communicate with modern devices?
